src/v2/components/Validators/Detail/index.jsx

```diff
@@ -3,41 +3,24 @@ import {Container, useTheme} from '@material-ui/core';
 import {observer} from 'mobx-react-lite';
 import useMediaQuery from '@material-ui/core/useMediaQuery/useMediaQuery';
 import React, {useEffect} from 'react';
-import {map, find, eq} from 'lodash/fp';
+import {map, find} from 'lodash/fp';
 import {Match} from 'react-router-dom';
-import getUptime from 'v2/utils/getUptime';
 
-import {
-  ComposableMap,
-  Geographies,
-  Geography,
-  Marker,
-  Markers,
-  ZoomableGroup,
-} from 'react-simple-maps';
+import getUptime from 'v2/utils/getUptime';
 import SectionHeader from 'v2/components/UI/SectionHeader';
 import NodesStore from 'v2/stores/nodes';
-import OverviewStore from 'v2/stores/networkOverview';
-import {mapStyle, markerStyle} from 'v2/theme';
-import MapTooltip from 'v2/components/UI/MapTooltip';
 import HelpLink from 'v2/components/HelpLink';
 import Button from 'v2/components/UI/Button';
 import Avatar from 'v2/components/UI/Avatar';
 import Mixpanel from 'v2/mixpanel';
 import CopyBtn from 'v2/components/UI/CopyBtn';
 
 import {LAMPORT_SOL_RATIO} from '../../../constants';
+import ValidatorsMap from '../../ValidatorsMap';
```

Comment:
should these be `v2/...` paths instead of relative? I prefer that kind, but interested in your thoughts (figure we should standardize since we use `v2/...` above